package.json

```diff
@@ -10,6 +10,9 @@
         "lint": "next lint"
     },
     "dependencies": {
+        "@emotion/styled": "^11.9.3",
+        "@material-ui/icons": "^4.11.3",
+        "@mui/icons-material": "^5.8.4",
```

Comment:
Instead of using the core MUI icon package, could we use something like [react-icons][https://react-icons.github.io/react-icons/] instead? In addition to a simpler API, it includes MUI icons along with other popular icon packages in case an icon is missing.

Reply:
Ok, I will see it